### PR TITLE
JENA-1994: Look in datatype URI

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: jena-core/src/test/java/org/apache/jena/util/TestPrefixMappingUtils.java [hunk 1]
```diff
@@ -141,14 +146,125 @@ static int size(PrefixMapping pmap) {
...
+             ) ;
+        
+        Graph graph1 = create(data2) ;
+        PrefixMapping pmap = PrefixMappingUtils.calcInUsePrefixMappingTTL(graph1) ;
+        PrefixMapping pmapExpected = new PrefixMappingImpl() ;
+        pmapExpected.setNsPrefix("", "http://example/") ;
+        pmapExpected.setNsPrefix("ex", "http://example/ex#") ;
+        Assert.assertEquals(2, size(pmap)) ;
+        Assert.assertTrue(sameMapping(pmapExpected, pmap)) ;
+        Assert.assertTrue(pmap.getNsPrefixURI("notinuse") == null) ;
```
Comment (kinow) [inline jena-core/src/test/java/org/apache/jena/util/TestPrefixMappingUtils.java:198]: Tests confirming the prefixes are available after the calc methods are called :+1:

**Review by kinow — COMMENTED** [review]
`process` and `processTTL` looks similar, but I couldn't think of any way to simplify further the code. LGTM :+1:

**Comment by afs** [comment]
> process and processTTL looks similar, but I couldn't think of any way to simplify further the code.

Yes - they do look the same and when I came back to them for this ticket I thought - "missed that last time". But they are not the same with differences throughout.

**Comment by afs** [comment]
(I meant to do the usual commit-merge but something has changed and I didn't get that option - user error)

**Pull request merged**